README.md

```diff
@@ -1,88 +1,130 @@
+Download this project and run `pod install` to install TwilioVideo.framework. Open VideoQuickStart.xcworkspace in Xcode.
 
-- [Screen Capturer](ScreenCapturerExample) - Shows how to use `TVIScreenCapturer` to capture the contents of a `UIView`, and how a custom `TVIVideoCapturer` can be implemented to do the same.
 
-## Prerequisites
 
-This project uses Apple's Swift programming language 3.0 for iOS, and the only supported way to develop iOS apps is on an Apple computer running OS X and Xcode. We have tested this application with the latest versions of iOS (10.0) and Xcode (8.0) at the time of this writing.
+## Quickstart
 
-You can use the iOS Simulator that comes with Xcode to do your testing, but local video will not be shared since the Simulator cannot access a camera. If you have an iOS device, you can now run apps from Xcode on your device without a paid developer account.
 
-## Twilio Library Setup for the Project
 
-You will need to add the Twilio Video library to the project to compile and run. You can do this using the [Cocoapods](https://cocoapods.org/) dependency management system. Version 1.0 or greater is required.
+### Running the Quickstart
 
-### Using Cocoapods
+To get started with the Quickstart application follow these steps:
 
-First, you will need to have Cocoapods 1.0.0+ installed on your Mac, so go ahead and do that if you haven't already - the directions are here: [Getting Started with Cocoapods](https://guides.cocoapods.org/using/getting-started.html). If you're not sure, type `pod --version` into a command line.
+1. Open this project in XCode
 
-Next, just run `pod install` from the command line in the top level directory of this project. Cocoapods will install the Twilio library and then set up a .xcworkspace file that you will use to run your project from now on.
+<img width="700px" src="images/quickstart/xcode-video-quickstart.png"/>
+
+2. Next, you need to generate a valid Access Token. Log in to the Twilio Console and go to the [Testing Tools page](https://www.twilio.com/user/account/video/dev-tools/testing-tools) shown below. Pick a name for your user identity (such as "Alice"), type it in the identity textbox and click "Generate Access Token". Copy the Access Token string that will appear in the gray box - we will need in the following step.
+
+
+<img width="700px" src="images/quickstart/generate_access_tokens.png"/>
+
+*Note: If you enter the Room Name, then you can retrict this user's access to the specified Room only. Ideally, you want to implement and deploy an Access Token server to generate tokens. You can read more about setting up your own Access Token Server in this [section](#setup-an-access-token-server). Read this [tutorial](https://www.twilio.com/docs/api/video/user-identity-access-tokens) to learn more about Access Tokens.*
+
+3. Paste the token you generated in the earlier step in the `ViewController.swift`.
+
+<img width="700px" src="images/quickstart/xcode-video-quickstart-token.png"/>
```

Comment:
I generally found the screenshots kind of hard to read. Is it possible to adjust the width or focus more on the content that we need to see?